packages/ecs-agent/ecs-agent.spec

```diff
@@ -296,7 +296,7 @@ mv %{vpccni_gorepo}-%{vpccni_gitrev}/vendor go-vendor/%{vpccni_gorepo}
 # ├── LICENSE.amazon-ecs-cni-plugins
 # ├── LICENSE.amazon-vpc-cni-plugins
 # ├── NOTICE
-# ├── THIRD-PARTY
+# ├── THIRD_PARTY.md
```

Comment:
I would like to double check that the contents have not changed meaningfully. I don't expect they have, but when paths and extensions change, its worth confirming nothing has changed under us.

Reply:
This has been validated, the contents are same as earlier.